telegram_bot.py

```diff
@@ -226,6 +227,18 @@ def touch_liveness_probe():
     config["LIVENESS"].touch()
 
 
+def escape_md_v2(text):
```

Comment:
In order to save the definition of this function, we can perhaps use HTML instead of Markdown? Thanks to that, we could use this function https://docs.python.org/3/library/html.html#html.escape.